packages/driver/src/cypress/proxy-logging.ts

```diff
@@ -283,7 +283,7 @@ export default class ProxyLogging {
   }
 
   /**
-   * The `cy.route()` XHR stub functions will log before a proxy log is received, so this queues an XHR log to be overridden by a proxy log later.
+   * The `cy.intercept()` XHR stub functions will log before a proxy log is received, so this queues an XHR log to be overridden by a proxy log later.
```

Comment:
Now that we aren't overriding the XMLHttpRequest constructor and using it to trigger new Cypress.logs, this PR can get rid of this `addXhrLog` function, as well as any of the other logic in this file that specifically applies to XHRs.

A lot of the logic in this file is only there because there is a race condition between whether the XHR log is created first or the proxy request is created first, so this should simplify it a good deal.

Reply:
I removed `addXhrLog` and any logic pertaining to `UnmatchedXhrLog`. Is there anything else I should remove?

Reply:
Here are the other places I'm aware of: https://github.com/cypress-io/cypress/commit/eac156ec69409f24e5896088e584f256312186a7 Feel free to cherry-pick this in, I didn't just wanna commit to your branch directly, feels rude :laughing:

Reply:
not rude at all! Thanks for the help. I cherry picked your changes into this PR.